<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainQuery</class>
 <widget class="QMainWindow" name="TrainQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1690</width>
    <height>1250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_window">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color:rgb(206, 206, 206);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>1180</y>
         <width>112</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(255, 135, 163)
}</string>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>760</y>
         <width>201</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(243, 255, 189)
}</string>
       </property>
       <property name="text">
        <string>火车视觉识别</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>610</y>
         <width>201</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(243, 255, 189)
}</string>
       </property>
       <property name="text">
        <string>下料机控制</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>460</y>
         <width>201</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(243, 255, 189)
}</string>
       </property>
       <property name="text">
        <string>下料历史查询</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>201</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(243, 255, 189)
}</string>
       </property>
       <property name="text">
        <string>火车型号尺寸</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_menu">
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color:rgb(225, 255, 253)
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="System_name">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>721</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>19</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>火车型号尺寸 信息管理系统</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>580</y>
         <width>1351</width>
         <height>481</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextBrowser{
background-color:rgb(255, 255, 255)
}</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;******************** Train Model Table 数据 ********************&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; TrainTypeID  HoldType  ExLength  ExWidth  ExHeight  InLength  InWidth  InHeight  Volume  Density  LoadWeight  FullLength  BottomHeight HookType&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        1001      Coal      20.5      3.2       4.5      18.5      2.8       4.0   200.0      0.8       150.0        25.0           1.2   Type-A       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        1002      Coal      22.0      3.5       4.8      20.0      3.0       4.3   250.0      0.8       180.0        27.0           1.3   Type-B       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        1003      Coal      18.0      3.0       4.0      16.0      2.5       3.5   160.0      0.8       120.0        23.0           1.1   Type-A       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        2001     Mixed      25.0      3.8       5.0      23.0      3.5       4.5   300.0      0.7       200.0        30.0           1.5   Type-C       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        2002     Mixed      24.0      3.6       4.8      22.0      3.3       4.3   280.0      0.7       190.0        29.0           1.4   Type-C       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        3001 Container      15.0      2.8       3.5      14.0      2.5       3.2   120.0      NaN         NaN        20.0           1.0   Type-D       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        3002      Bulk      19.0      3.1       4.2      17.0      2.8       3.8   180.0      0.9       140.0        24.0           1.2   Type-A       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        3003    Tanker      16.0      2.9       3.8      15.0      2.6       3.5   140.0      NaN         NaN        21.0           1.1   Type-E       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        4001      Coal      21.0      3.3       4.6      19.0      2.9       4.1   220.0      0.8       160.0        26.0           1.3   Type-B       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        4002     Mixed      23.0      3.7       4.9      21.0      3.4       4.4   290.0      0.7       195.0        28.0           1.4   Type-C       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        4003      Coal      17.0      2.9       3.9      15.5      2.6       3.6   150.0      0.8       110.0        22.0           1.0   Type-A       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        4004      Bulk      20.0      3.2       4.4      18.0      2.9       4.0   210.0      0.9       155.0        25.5           1.3   Type-B       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        5001 Container      14.0      2.7       3.3      13.0      2.4       3.0   110.0      NaN         NaN        19.0           0.9   Type-D       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        5002    Tanker      17.0      3.0       4.0      16.0      2.7       3.6   160.0      NaN         NaN        22.0           1.1   Type-E       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        5003      Coal      19.5      3.1       4.3      17.5      2.8       3.9   190.0      0.8       140.0        24.5           1.2   Type-A       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        6001     Mixed      26.0      3.9       5.2      24.0      3.6       4.7   320.0      0.7       210.0        31.0           1.6   Type-C       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        6002      Bulk      18.0      3.0       4.1      16.5      2.7       3.7   170.0      0.9       130.0        23.0           1.1   Type-B       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        6003      Coal      20.8      3.4       4.7      19.0      3.1       4.2   230.0      0.8       170.0        26.5           1.4   Type-B       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        7001 Container      16.5      3.0       3.7      15.5      2.7       3.4   150.0      NaN         NaN        21.5           1.1   Type-D       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        7002    Tanker      18.0      3.2       4.2      17.0      2.9       3.8   190.0      NaN         NaN        23.0           1.2   Type-E       &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;*********************************************************&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_qy">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>280</y>
         <width>200</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>火车型号ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ls">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>1070</y>
         <width>251</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>数据库结果</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>300</y>
         <width>701</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit{
	background-color:rgb(255, 240, 226)
}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>440</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(213, 227, 255)
}</string>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>440</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(213, 227, 255)
}</string>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_qy_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>170</y>
         <width>391</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>请输入查询/删除的信息：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_11">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>1160</y>
         <width>291</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(213, 227, 255)
}</string>
       </property>
       <property name="text">
        <string>新增/修改</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
